Ens

subramanianv opened the pull request.

villanuevawill commented on src/components/Avatar/index.js:
```diff
@@ -95,6 +116,7 @@ Avatar.propTypes = {
   nameTextWeight: PropTypes.string,
   addressTextScale: PropTypes.string,
   addressTextColor: PropTypes.string,
+  ensDomain: PropTypes.string,
```
Awesome - the addition to the component is exactly right. 👍

villanuevawill commented:
@subramanianv in general this looks great. I'm going to run all this on staging tomorrow and make sure it isset.

villanuevawill commented:
@subramanianv there are a few issues:
1. ensdomain is not showing up on the bounty list page for the profile
2. It is not showing up on the profile page for the bounty list
3. Also not showing up on leaderboard

villanuevawill commented:
Other issue is it only needs to passed through into the avatar locations that address is being passed through - otherwise it's redundant.

subramanianv commented:
@villanuevawill 

> @subramanianv there are a few issues:
> 
> 1. ensdomain is not showing up on the bounty list page (/explorer)
     Fixed in the latest push
> 2. It is not showing up on the profile page for the bounty list (/profile/hash)
    Fixed in the latest push
> 3. Also not showing up on leaderboard (/leaderboard)
    I think we need to change the leaderboard query on API and also include the ens domain on the leaderboard serializer
[https://github.com/subramanianv/BountiesAPI/commit/f04635ce1801c0397ed6952af3e2c13ece0025a5](commit)
Will open up a pr for that

villanuevawill commented:
@subramanianv just waiting on the leaderboard PR... otherwise this is looking good

subramanianv commented:
@villanuevawill Here is the leaderboard
https://github.com/Bounties-Network/BountiesAPI/pull/242
